tests/infra/consortium.py

```diff
@@ -285,9 +303,12 @@ def vote_using_majority(
                 response = member.vote(remote_node, proposal, ballot)
                 if response.status_code != http.HTTPStatus.OK.value:
                     raise infra.proposal.ProposalNotAccepted(proposal, response)
-                proposal.state = infra.proposal.ProposalState(
-                    response.body.json()["state"]
-                )
+                body = response.body.json()
+                for k in ("state", "proposalState"):
+                    if k in body:
+                        proposal_state = body[k]
+                        break
```

Comment:
```suggestion
                proposal_state = body.get("state", body.get("proposalState"))
                assert proposal_state, f"Could not find proposal state in {body}"
```

Seems marginally nicer